browser-test/src/support/admin_programs.ts

```diff
@@ -216,6 +222,17 @@ export class AdminPrograms {
     await this.expectDraftProgram(programName);
   }
 
+  async createPublicVersion(programName: string) {

```

Comment:
nit: It's nice to separate expects and test setup with empty lines -- we used this as style best practices on my last team. This way, we can see clearly what part of this function is general setup (clicking buttons, navigating to pages) and what part is actually expecting something.

So for example, I would add some empty lines to this function to make it look like:
async createPublicVersion(){
    wait this.gotoAdminProgramsPage();

    await this.expectActiveProgram(programName);

    await this.page.click(this.selectWithinProgramCard(programName, 'ACTIVE', ':text("New Version")'));
    await waitForPageJsLoad(this.page);
    await this.page.check(`label:has-text("Public")`);
    await this.page.click('#program-update-button');
    await waitForPageJsLoad(this.page);

    await this.expectDraftProgram(programName);
}

Reply:
Thanks, made that change